setup.py

```diff
@@ -41,12 +41,15 @@
         'matplotlib',
         'pandas',
         'h5py',
-        'scanpy>=1.5.1,<1.6.0',
```

Comment:
do we plan to do a release with 1.5.x or will we just skip it?

Reply:
Ahh- actually I should make this >=1.6.0- thanks.

Reply:
Some of the changes here are 1.6.0-specific, so will need this pin. I released a 1.5.1 compatible version yesterday (though without the batch stuff implemented here).